files/image_config/procdockerstats/procdockerstats

```diff
@@ -154,6 +155,7 @@ class ProcDockerStats:
         for k1,v1 in dockerdata.iteritems():
             for k2,v2 in v1.iteritems():
                 self.update_state_db(k1, k2, v2)
+                return True
```

Comment:
Should this return statement be inside the inner for loop here? The loop will only execute once. I believe you want this line outside _both_ for loops.